web-client/src/modules/requests/containers/PostsContainer.tsx

```diff
@@ -129,7 +129,6 @@ const OfferAndRequestPostsContainer: React.FC<PostsProps> = ({
 };
 
 interface PostsProps {
-  postMode: ApplicationPreference; // offer or request
-  status: string;
+  postType: string; // offer or request
```

Comment:
Since the container expects the value of this prop to be one of two things, it's better to have it as an enum so as to leverage typescript features and to avoid confusion for developers